runtime-data/wsl-default-settings/autobuild/overrides/usr/lib/wsl/wsl-oobe.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -ue
+
+DEFAULT_GROUPS='audio,cdrom,plugdev,video,wheel'
+DEFAULT_UID='1000'
+
+echo 'Please create a default UNIX user account. The username does not need to match your Windows username.'
+echo 'For more information visit: https://aka.ms/wslusers'
+
+if getent passwd "$DEFAULT_UID" > /dev/null ; then
+  echo 'User account already exists, skipping creation'
+  exit 0
+fi
+
+while true; do
+
+  # Prompt from the username
+  read -p 'Enter new UNIX username: ' username
+
+  # Create the user
+  if /usr/sbin/useradd --create-home --shell /bin/bash --uid "$DEFAULT_UID" --quiet --comment ''  "$username"; then
+
+    if /usr/sbin/usermod "$username" -aG "$DEFAULT_GROUPS"; then
```

Comment:
Indentation seems messed up.

Reply:
That was from Microsoft's sample file, no idea how it should be.